.dockerignore

```diff
@@ -0,0 +1,15 @@
+__pycache__
+*.pyc
+*.pyo
+*.pyd
+*.db
+*.env
+README.md
+Dockerfile
```

Comment:
Including Dockerfile in .dockerignore is unusual if you are building the image using this Dockerfile. Ensure this is intentional and aligns with your build process.